apis/pkg/v1/interfaces.go

```diff
@@ -46,6 +53,18 @@ func RefNames(refs []corev1.LocalObjectReference) []string {
 	return stringRefs
 }
 
+// GetPackageOwnerReference returns the owner reference that points to the owner
+// package of given revision, if it can find one.
+func GetPackageOwnerReference(rev resource.Object) (metav1.OwnerReference, bool) {
```

Comment:
I thought of having `GetPackageOwnerReference` and `SetPackageOwnerReference` for convenience but realized we use `resource.Object` in the places I need these functions and didn't want to do a big change. So I opted for making it a helper function here.

Reply:
I like this approach of using a helper function, but I'd prefer it to live near where it's used in the controller code rather than with the API type definitions.

Reply:
I felt like it could be useful in other contexts as well but we can move it there once we know that's needed. Just moved it to controller and added tests.